### ISSUE TITLE
CheckBox and CheckBoxGroup - children stories should have focus indicator

### ISSUE THREAD

**Issue by jcfilben** [grommet/grommet#7451]
### Expected Behavior
A focus indicator should be shown when navigating with keyboard on the CheckBox for the CheckBox/Children and CheckBoxGroup/Children stories

### Actual Behavior
Focus indicator is not displayed

### URL, screen shot, or Codepen exhibiting the issue
https://storybook.grommet.io/?path=/story/input-checkbox-children--children
https://storybook.grommet.io/?path=/story/input-checkboxgroup-children--children

### Steps to Reproduce

1. 2. 3.

### Your Environment

- Grommet version:
- Browser Name and version:
- Operating System and version (desktop or mobile):
